src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig

```diff
@@ -49,6 +49,23 @@ Closing: 17:00</p>
 
 <p>Parts of the workshop will also be accessible remotely. More info soon.</p>
 
+<h2>Call for Proposals</h2>
+
+<p>Please submit your ideas for presentations, lightning talks or panels at https://bit.ly/nttw3proposals (where you can see ideas already proposed) or submit ideas for the program <a href="{{ path('ma_contact') }}">by emailing the conference organizers directly</a>. We’d like to encourage presentations from a variety of perspectives related to our themes of:</p>
```

Comment:
Argh, I missed it before merge, but the link is not clickable (see website, it has the update)